sql/mysqld.cc

```diff
@@ -11838,8 +11838,13 @@ static int get_options(int *argc_ptr, char ***argv_ptr) {
 static void set_server_version(void) {
   char *end [[maybe_unused]] = strxmov(server_version, MYSQL_SERVER_VERSION,
                                        MYSQL_SERVER_SUFFIX_STR, NullS);
+#ifdef HAVE_PERCONA_PROBUILD
+  if (!strstr(MYSQL_SERVER_SUFFIX_STR, "-pro"))
+    end = my_stpcpy(end, "-pro");
+#endif
```

Comment:
This change is not required if we define this in CMake and the above code will take care of this